src/amo/components/CategoryHead/index.js

```diff
@@ -0,0 +1,99 @@
+/* @flow */
+import config from 'config';
+import invariant from 'invariant';
+import * as React from 'react';
+import Helmet from 'react-helmet';
+import { connect } from 'react-redux';
+import { compose } from 'redux';
+
+import { getCanonicalURL } from 'amo/utils';
+import { ADDON_TYPE_EXTENSION, ADDON_TYPE_THEME } from 'core/constants';
+import translate from 'core/i18n/translate';
+import type { AppState } from 'amo/store';
+import type { I18nType } from 'core/types/i18n';
+
+type CategoryType = {|
+  application: string,
+  description: string,
+  id: number,
+  misc: boolean,
+  name: string,
+  slug: string,
+  type:
+    | 'extension'
+    | 'theme'
+    | 'dictionary'
+    | 'search'
+    | 'language'
+    | 'persona',
```

Comment:
do we need all of these types? on the current category pages, I only show extension and persona? let me know if I am missing something..

Reply:
The API returns more categories than that, I, listed all thd types mentioned in https://addons-server.readthedocs.io/en/latest/topics/api/categories.html#get--api-v4-addons-categories-